packages/tupaia-web/src/features/Visuals/View/Download/DownloadFiles.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+import React, { useState } from 'react';
+import { ViewConfig, ViewReport } from '@tupaia/types';
+import { useDownloadFiles } from '../../../../api/mutations';
+import { DownloadVisual } from './DownloadVisual';
+
+interface DownloadFilesVisualProps {
+  report: ViewReport;
+  config: ViewConfig;
+  isEnlarged?: boolean;
+}
+
+export const DownloadFiles = ({ report, isEnlarged }: DownloadFilesVisualProps) => {
+  const { mutateAsync: download, error, reset, isLoading } = useDownloadFiles();
+  const [isDownloading, setIsDownloading] = useState(false);
```

Comment:
Out of interest, what is the difference between isDownloading and isLoading that is returned from useDownloadFiles?